### PR TITLE
Disable go button when directions are not populated

### PR DESCRIPTION
**Pull request by matbmapspeople**
What: 

Disable 'Go' button if directions are not populated.

How: 
Created a simple boolean that is false when we start fetching data.
When data is populated, the boolean is set to true.
Applied specific styling for disabled button.

Working example:

https://github.com/MapsPeople/web-ui/assets/56337591/65377719-32f7-4763-b234-b8747877aaa7

PS. Talking to @andreeaceachir142, I removed two input fields for the time data is being fetched.

### CODE REVIEW

File: packages/map-template/src/components/Wayfinding/Wayfinding.jsx [hunk 1]
```diff
@@ -479,15 +479,15 @@ function Wayfinding({ onStartDirections, onBack, directionsToLocation, direction
                         {travelMode === travelModes.DRIVING && <DriveIcon />}
                         {travelMode === travelModes.BICYCLING && <BikeIcon />}
                         <div>{t('Distance')}:</div>
-                        <div className="wayfinding__meters">{totalDistance && <mi-distance unit={distanceUnitSystem} meters={totalDistance} />}</div>
+                        <div className="wayfinding__meters">{totalDistance && isDirectionReady && <mi-distance unit={distanceUnitSystem} meters={totalDistance} />}</div>
```
Comment (ammapspeople) [inline packages/map-template/src/components/Wayfinding/Wayfinding.jsx:482]: Is this change necessary? The `<mi-distance>` seems to be showing nothing anyway.

File: packages/map-template/src/hooks/useDirectionsInfo.js [hunk 2]
```diff
@@ -5,16 +5,18 @@ import directionsResponseState from '../atoms/directionsResponseState';
...
- * Hook to handle when both origin location and destination location are selected, 
+ * Hook to handle when both origin location and destination location are selected,
  * and have geometry, call the MapsIndoors SDK to get information about the route.
  */
 const useDirectionsInfo = (originLocation, destinationLocation, directionsService, travelMode, accessibilityOn) => {
     const [totalDistance, setTotalDistance] = useState()
     const [totalTime, setTotalTime] = useState();
     const [hasFoundRoute, setHasFoundRoute] = useRecoilState(hasFoundRouteState);
     const [, setDirectionsResponse] = useRecoilState(directionsResponseState);
+    const [isDirectionReady, setIsDirectionReady] = useState()
```
Comment (ammapspeople) [inline packages/map-template/src/hooks/useDirectionsInfo.js:16]:
```suggestion
    const [areDirectionsReady, setAreDirectionsReady] = useState()
```
We usually use the plural form, "directions".

Comment (matbmapspeople) [reply packages/map-template/src/hooks/useDirectionsInfo.js:16]: Done ✅

File: packages/map-template/src/components/Wayfinding/Wayfinding.jsx (hunk 1 above)
Comment (matbmapspeople) [reply packages/map-template/src/components/Wayfinding/Wayfinding.jsx:482]: It does change when you f. ex. switch directionsTo to Viborg. Without it, for a few seconds the time/distance is shown from the previous search. It's a lil-UX improvement.

Comment (ammapspeople) [reply packages/map-template/src/components/Wayfinding/Wayfinding.jsx:482]: 👏

File: packages/map-template/src/hooks/useDirectionsInfo.js (hunk 2 above)
Comment (ammapspeople) [reply packages/map-template/src/hooks/useDirectionsInfo.js:16]: I still see `isDirectionReady` in the code.

Comment (matbmapspeople) [reply packages/map-template/src/hooks/useDirectionsInfo.js]: Please try now. I might have not pushed changes for some reason @ammapspeople

**Comment by matbmapspeople** [comment]
Done ✅ @ammapspeople

**Comment by matbmapspeople** [comment]
Hey @andreeaceachir142 
Can you please take a final look so we can proceed with it?

**Comment by andreeaceachir142** [comment]
Approved ✅ Can you please write some test cases in the ticket so I can test out the feature? @matbmapspeople
